lib/bolt/bolt_option_parser.rb

```diff
@@ -156,18 +156,30 @@ def get_help_text(subcommand, action = nil)
       end
     end
 
+    COLORS = {
+      cyan: "36"
+    }.freeze
+
+    def self.colorize(color, string)
+      if $stdout.isatty
+        "\033[#{COLORS[color]}m#{string}\033[0m"
+      else
+        string
+      end
+    end
+
     BANNER = <<~HELP
-      NAME
+      #{colorize(:cyan, 'NAME')}
           bolt
```

Comment:
```suggestion
      #{colorize(:cyan, 'Name')}
        bolt
```

I'm guessing colorizing the headers is to bring this more in line with the show commands? If so, we should use sentence casing for the headers and decrease the indent for the rest of the text to 2 instead of 4.